pkg/api-server/policies_endpoints.go

```diff
@@ -0,0 +1,36 @@
+package api_server
+
+import (
+	"sort"
+
+	"github.com/emicklei/go-restful/v3"
+
+	"github.com/kumahq/kuma/pkg/api-server/types"
+	"github.com/kumahq/kuma/pkg/config/core"
+	"github.com/kumahq/kuma/pkg/core/resources/model"
+)
+
+func addPoliciesWsEndpoints(ws *restful.WebService, mode core.CpMode, readOnly bool, defs []model.ResourceTypeDescriptor) error {
+	ws.Route(ws.GET("/policies").To(func(req *restful.Request, resp *restful.Response) {
+		response := types.PoliciesResponse{}
+		for _, def := range defs {
+			if !def.IsPolicy {
+				continue
+			}
+			response.Policies = append(response.Policies, types.PolicyEntry{
+				Name:        string(def.Name),
+				ReadOnly:    readOnly || core.Zone == mode,
```

Comment:
do we plan this to be different for different policies? if not, maybe this should be top level next to `policies`?

Reply:
I'm not sure I felt that this was just more flexible for the future (likely if we deprecate things for example). ATM the whole policy engine enables this.

Reply:
Actually adding `def.ReadOnly` too :)